zellij-tile/src/data.rs

```diff
@@ -24,6 +24,17 @@ pub enum Key {
     Esc,
 }
 
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+
+pub enum Mouse {
+    ScrollUp(usize),   // number of lines
+    ScrollDown(usize), // number of lines
+    ClearScroll,
+    LeftClick(isize, usize),              // line and column
```

Comment:
I'm curious about the `isize` here. If possible, I'd like to make everything `(usize, usize)` :)